demos/python_demos/text_spotting_demo/text_spotting_demo.py

```diff
@@ -221,7 +206,22 @@ def main():
     text_enc_exec_net = ie.load_network(network=text_enc_net, device_name=args.device)
     text_dec_exec_net = ie.load_network(network=text_dec_net, device_name=args.device)
 
-    hidden_shape = text_dec_net.inputs[args.trd_input_prev_hidden].shape
+    for name, input in text_dec_exec_net.inputs.items():
+        if len(input.shape) == 3:
+            if input.shape[1] == 1:
+                trd_input_prev_hidden = name
+            else:
+                trd_input_encoder_outputs = name
+        elif len(input.shape) == 1:
+            trd_input_prev_symbol = name
+
+    for name, output in text_dec_exec_net.outputs.items():
+        if len(output.shape) == 3:
+           trd_output_cur_hidden = name
+        elif len(output.shape) == 2:
+           trd_output_symbols_distr = name
+
+    hidden_shape = text_dec_net.inputs[trd_input_prev_hidden].shape
```

Comment:
I think it can be useful to check that each of `trd_input_prev_hidden`, `trd_input_encoder_outputs`, `trd_input_prev_symbol`, `trd_output_cur_hidden` and `trd_output_symbols_distr` vars are found and initialized.

Reply:
added such check